pages/source-code.html

```diff
@@ -0,0 +1,200 @@
+                </li>
+                <!-- FAQ -->
+                <li class="nav-item">
+                    <a class="nav-link" href="../pages/faq.html">FAQ</a>
+                </li>
+                <!-- NEWS -->
+                <li class="nav-item">
+                    <a class="nav-link" href="../pages/news.html">NEWS</a>
+                </li>
+                <!-- Contact -->
+                <li class="nav-item">
+                    <a class="nav-link" href="../pages/contact.html">CONTACT</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="https://twitter.com/UyuniProject" target="_blank">
+                        <img src="../img/twitter.svg" alt="Uyuni - Twitter" title="Twitter">
+                    </a>
+                </li>
+            </ul>
+        </nav>
+        <!-- #nav -->
+        <div class="subpage-description">
+            <div class="subpage-title-back">
+                <h1>Contact</h1>
+                <a href="../index.html" class="btn btn-primary">
+                    <i class="eos-icons">keyboard_arrow_left</i> Go back</a>
+            </div>
+        </div>
+    </div>
+</header>
+
+<div class="container page-wrapper">
+    <div class="container">
+        <div class="row content-padding subpage-content">
+            <h4><a href="Introduction">Introduction</a></h4>
+            <p>We maintain the source code in three places:</p>
+            <ul>
+                <li>Uyuni Project</li>
+                <li>External Git repositories</li>
+                <li>OpenBuildService</li>
+            </ul>
+            <h4><a href="#uyuni-project">Uyuni Project</a></h4>
+            <p>This is the <a href="https://github.com/uyuni-project" target="_blank">main GitHub project</a> that contains most of the source code.</p>
+            <p>The following repositories are maintained:</p>
+            <ul>
+                <li><a href="https://github.com/uyuni-project/uyuni" target="_blank">uyuni</a>: Contains the main Uyuni source code (including, for example the source code for bakend, frontend, database or cucumber testsuite).</li>
+                <li><a href="https://github.com/uyuni-project/uyuni-docs" target="_blank">uyuni-docs</a>: The Uyuni documentation.</li>
+                <li><a href="https://github.com/uyuni-project/retail" target="_blank">retail</a>: The Uyuni documentation.</li>
+                <li><a href="https://github.com/uyuni-project/hub-xmlrpc-api" target="_blank">hub-xmlrpc-api</a>: The hub-xmlrpc-api package that allows a Server of Serves architecture (Hub).</li>
+                <li><a href="https://github.com/uyuni-project/uyuni-rfc" target="_blank">uyuni-rfc</a>: Uyuni feature RFCs.</li>
+                <li><a href="https://github.com/uyuni-project/uyuni-project.github.io" target="_blank">uyuni-project.github.io</a>: Uyuni Retail salt formulas and related tools and documentation.</li>
```

Comment:
`Uyuni Retail salt formulas and related tools and documentation.` This does not sound correct.

Reply:
Oops... yeah. Wrong copy paste.